bookkeeper-server/src/main/java/org/apache/bookkeeper/client/LedgerChecker.java

```diff
@@ -159,6 +166,8 @@ public LedgerChecker(BookieClient client, BookieWatcher watcher, int inFlightRea
         } else {
             semaphore = null;
         }
+        executor = Executors.newFixedThreadPool(Runtime.getRuntime().availableProcessors(),
```

Comment:
Do we need `Runtime.getRuntime().availableProcessors()` threads to deal with BookieLedgerChecker callback?

Reply:
@hangc0276 We don't necessarily need threads equal to the number of processors. I thought it was strange to specify a fixed value, so I just set `Runtime.getRuntime().availableProcessors()`.